crates/astria-sequencer/src/app.rs

```diff
@@ -639,6 +639,10 @@ impl App {
             .get_block_deposits()
             .await
             .context("failed to get block deposits in end_block")?;
+        debug!(
+            deposits = %telemetry::display::json(&DisplayDeposits(&deposits)),
+            "end_block: got block deposits"
```

Comment:
The `end_block:` is not necessary. You already running under a scope called `App::end_block`
```suggestion
            "read block deposits from state"
```